addons/project_timesheet_holidays/models/resource_calendar_leaves.py

```diff
@@ -199,6 +199,38 @@ def _generate_timesheeets(self):
         if results_with_leave_timesheet:
             results_with_leave_timesheet._timesheet_create_lines()
 
+    def _generate_public_time_off_timesheets(self, employees):
+        timesheet_vals_list = []
+        work_hours_data = self._work_time_per_day()
+        timesheet_read_group = self.env['account.analytic.line']._read_group(
+            [('global_leave_id', 'in', self.ids), ('employee_id', 'in', employees.ids)],
+            ['date:array_agg'],
+            ['employee_id']
```

Comment:
The `_read_group` has changed in that version. 

```suggestion
            ['employee_id'],
            ['date:array_agg'],
```